lib/tools/Curl.class.php

```diff
@@ -27,16 +28,18 @@ public static function put($url, $params = [], $options = [])
   public static function doCurl($method, $url, $params = [], $options = [])
   {
     $defaults = [
+      'cache'            => false,
       'headers'          => [],
       'verify'           => true,
-      'timeout'          => 5,
+      'timeout'          => 10, //5 was timing out on /roadmap for Asana API
```

Comment:
the default should still be 5 (or even lower). you can bump the option up to 10 for asana requests if you want